StateSpecs-DSL-ShouldExpressions-Tests.package/SpecOfShouldExpressionTests.class/instance/testReturningValidationResults.st

```diff
@@ -0,0 +1,20 @@
+tests
+testReturningValidationResults
+	| results |
+	self shouldnt: [
+		results := [ {
+			true should be: true.
+			42 should be: 'answer to life'.
+			42 where even should be: true.
+			[ 1/0 ] should raise: ZeroDivide.
+			[ false should be: true ] should raise: SpecOfFailed
+			}
+		] acceptingStateSpecsFailures
```

Comment:
maybe better to have value prefix for consistency with other evaluation methods:
```Smalltalk
[] valueAcceptingStateSpecsFailures
```

Reply:
yup… I also hesitated with *ignoring* or *silencing* instead of *accepting*, which might make more sense… or `valueWithBenign...Failures`

Reply:
yeah…

Reply:
I'm ok with not adding it; in fact in Clap I'm just using `#on:do:`…